src/Nethermind/Nethermind.Network.Test/ForkInfoTests.cs

```diff
@@ -94,8 +98,10 @@ public void Fork_id_and_hash_as_expected_on_rinkeby(long head, string forkHashHe
         [TestCase(6_485_845L, "0xd6e2149b", 6_485_846L, "Last Petersburg block")]
         [TestCase(6_485_846L, "0x4bc66396", 7_117_117L, "First Istanbul block")]
         [TestCase(7_117_117L, "0x6727ef90", 9_812_189L, "First Muir Glacier block")]
-        [TestCase(9_812_189L, "0xa157d377", 0L, "First Berlin block")]
-        [TestCase(9_900_000L, "0xa157d377", 0L, "Future Berlin block")]
+        [TestCase(9_812_189L, "0xa157d377", 10_499_401L, "First Berlin block")]
+        [TestCase(9_900_000L, "0xa157d377", 10_499_401L, "Future Berlin block")]
+        [TestCase(10_499_401L, "0x7119B6B3", 0L, "First London block")]
+        [TestCase(11_399_301L, "0x7119B6B3", 0L, "Future London block")]
```

Comment:
11399301 looks so specific that it may cause the reader to think that it has some meaning